src/ClaimService.Business/Features/Claims/Commands/Edit/EditClaimValidator.cs

```diff
@@ -160,14 +166,54 @@ public class EditClaimValidator : BaseEditRequestValidator<EditClaimRequest>, IE
       });
 
     #endregion
+
+    #region ResponsibleUserId
+
+    await AddFailureForPropertyIfAsync(
+      nameof(EditClaimRequest.ResponsibleUserId),
+      x => x == OperationType.Replace,
+      new()
+      {
+        {
+          async (x) => x.value is not null && Guid.TryParse(x.value.ToString().Trim(), out Guid userId) &&
```

Comment:
if this id is null, validator returns BadRequest, but accordind to request model ResponsibleUserId may be null